chalk-ir/src/lib.rs

```diff
@@ -642,7 +637,7 @@ pub struct Fn<I: Interner> {
 
 #[derive(Copy, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, HasInterner)]
 pub struct Lifetime<I: Interner> {
-    interned: I::InternedLifetime,
+    pub interned: I::InternedLifetime,
```

Comment:
Why did you make this pub? You can use `interned()`

Reply:
Some code in `ChalkIr` referenced it directly. I'll change it to the method call.